src/cli.rs

```diff
@@ -750,6 +750,11 @@ pub struct RenewArgs {
     #[clap(long, action, conflicts_with("crate-name"))]
     pub expiring: bool,
 
+    /// Renew wildcard audits for inactive crates which have not been updated
+    /// in 4 months.
+    #[clap(long, action, requires("expiring"))]
+    pub include_inactive: bool,
```

Comment:
Maybe we want this to optionally indicate the inactive period (to override the default 4 months)?

Reply:
I imagine we'd would want that to be a separate flag to specify the inactive period to ignore, as this flag currently completely eliminates the inactive period (which I guess would be ~equivalent to setting it to ∞ years). I'm not sure how much value something like that would have, so I'm a bit inclined to not add it unless we decide it's necessary.

If we did have a way to configure the inactive period, I imagine it would be more useful as a field in `config.toml` or similar, so that it would also impact the warnings from `cargo vet`.